scrape/scrape.go

```diff
@@ -826,8 +827,11 @@ type scrapeLoop struct {
 	labelLimits              *labelLimits
 	interval                 time.Duration
 	timeout                  time.Duration
-	scrapeClassicHistograms  bool
-	enableCTZeroIngestion    bool
+
+	// Feature flagged options.
+	scrapeClassicHistograms        bool
```

Comment:
Actually `scrapeClassicHistograms` comes from scrape config, so should be kept in place.